vm/systemSmartContracts/auction.go

```diff
@@ -471,7 +467,7 @@ func (s *stakingAuctionSC) registerBLSKeys(
 ) ([][]byte, error) {
 	maxNodesToRun := big.NewInt(0).SetBytes(args[0]).Uint64()
 	if uint64(len(args)) < maxNodesToRun+1 {
-		log.Debug("not enough arguments to process stake function")
+		s.eei.AddReturnMessage("not enough arguments to process stake function")
```

Comment:
We can use the same pattern used in this file:
s.eei.AddReturnMessage(fmt.Sprintf("not enough arguments to process stake function: expected min %d, got %d", maxNodesToRun+1, len(args)))

Reply:
done